packages/suite/src/views/wallet/transactions/TradeBox/TradeBoxPrices.tsx

```diff
@@ -0,0 +1,132 @@
+import { localizeNumber } from '@suite-common/wallet-utils';
+import { Icon, useTheme } from '@trezor/components';
+import { FONT_SIZE, FONT_WEIGHT, SCREEN_SIZE } from '@trezor/components/src/config/variables';
+import { PropsWithChildren, ReactNode, useEffect, useState } from 'react';
+import { FiatValue, Translation } from 'src/components/suite';
+import { useSelector } from 'src/hooks/suite';
+import { Account } from 'src/types/wallet';
+import {
+    TradeBoxGraph,
+    TradeBoxGraphProps,
+} from 'src/views/wallet/transactions/TradeBox/TradeBoxGraph';
+import styled from 'styled-components';
+
+const GRAPH_RATES_LAST_DAYS = 14;
+
+const Wrapper = styled.div`
+    display: flex;
+
+    & > *:not(:first-child) {
+        margin-left: 42px;
+
+        @media (max-width: ${SCREEN_SIZE.LG}) {
+            margin-left: 28px;
+        }
+
+        @media (max-width: 360px) {
+            margin-left: 16px;
+        }
+    }
+`;
+
+const Name = styled.div`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-size: ${FONT_SIZE.TINY};
+    margin-bottom: 3px;
+`;
+
+const Value = styled.div`
+    font-weight: ${FONT_WEIGHT.MEDIUM};
+`;
+
+const ChangeWrapper = styled.div<{ color: string }>`
+    display: flex;
+    align-items: center;
+    color: ${({ color }) => color};
+
+    > :first-child {
+        margin-right: 4px;
+    }
+`;
+
+interface TradeBoxHeadCardProps extends PropsWithChildren {
+    name: ReactNode;
+}
+
+const TradeBoxHeadCard = ({ name, children }: TradeBoxHeadCardProps) => (
+    <div>
+        <Name>{name}</Name>
+        <Value>{children}</Value>
+    </div>
+);
+
+const calculatePercentageDifference = (a: number, b: number) => ((a - b) / b) * 100;
+
+interface TradeBoxPricesProps {
+    account: Account;
+}
+
+export const TradeBoxPrices = ({ account }: TradeBoxPricesProps) => {
+    const theme = useTheme();
+    const locale = useSelector(state => state.suite.settings.language);
+    const localCurrency = useSelector(state => state.wallet.settings.localCurrency);
+    const rates = useSelector(state =>
+        state.wallet.fiat.coins.find(r => r.symbol === account?.symbol),
+    );
+
+    const lastTwoWeeksRates = rates?.lastTwoWeeks?.tickers ?? [];
+    const lastDayRate = lastTwoWeeksRates[0]?.rates?.[localCurrency];
+    const currentRate = rates?.current?.rates?.[localCurrency];
+    const rateGoingUp = currentRate && lastDayRate ? currentRate >= lastDayRate : false;
```

Comment:
same as `Ticker` pretty much, so some util could be made

Reply:
what you mean, util to compare two numbers? :D

Reply:
Yep, to get `rateGoingUp` based on current rates and rated from the past.

Reply:
Maybe in the next iteration of this task? I don't want to do any futher changes until https://github.com/trezor/trezor-suite/pull/9292 is done